src/steps/05-PerformanceRequirements/DOW/ArchitecturalDesign.vue

```diff
@@ -15,15 +15,45 @@
               and request CSPs to propose a customized cloud solution based on your unique
               objectives.
             </p>
-
+            <div v-if="noEnvNoXaaS() || hasEnvNoXaaS()">
+              <ATATAlert
+              id="ArchitecturalDesignAlert"
+              type="warning"
+              class="mb-10 mt-2"
+              :showIcon="true"
+            >
+              <template v-slot:content>
+                <p class="mr-5 mb-0 font-weight-400 font-size 16">
+                  Based on what you previously told us, we recommend selecting “Yes” below.
+                  If you don’t need an architectural design solution, you’ll need to revisit
+                  <span v-if="noEnvNoXaaS()">
+                    <router-link
```

Comment:
I think the logic is backward for showing/hiding the current functions link. When I say "NO" for current env in step 4, I see the link. When I say "YES" for current env in step 4, I don't see the link.